classes/install/Installer.php

```diff
@@ -875,15 +870,12 @@ public function columnExists($tableName, $columnName)
     /**
      * Check to see whether a table exists.
      * Used in installer XML in conditional checks on <data> nodes.
-     *
-     * @param string $tableName
-     *
-     * @return bool
      */
-    public function tableExists($tableName)
+    public function tableExists(string $tableName): bool
```

Comment:
This is only used in `classes/install/Upgrade.php` in `migrateStaticPagesToNavigationMenuItems`, which can be modified to use `Schema::hasTable` directly -- so I think this function can be removed.